src/App.js

```diff
@@ -4,14 +4,31 @@ import { Routes, Route } from "react-router-dom";
 import HomePage from "./pages/homepage/HomePage";
 import Login from "./pages/loginpage/LoginPage";
 import Signup from "./pages/signupPage/SignupPage";
+import History from "./pages/historyPage/History";
+import Playlist from "./pages/playlistPage/Playlist";
+import Watchlater from "./pages/watchlaterPage/Watchlater";
+import Likes from "./pages/likesPage/Likes";
+import SingleVideo from "./pages/singleVideoPage/SingleVideo";
+import PageNotFound from "./pages/pageNotFound/PageNotFound";
+import PlaylistVideos from "./pages/playlistVideos/PlaylistVideos";
```

Comment:
Suggestion: you can add index.js file in pages folder and import all files from there